pkg/controller/extension/shared/deployer_issuers.go

```diff
@@ -166,6 +167,13 @@ func (d *Deployer) createIssuer(input Issuer) *certv1alpha1.Issuer {
 			CA:   d.createCASpec(input),
 		},
 	}
+	if !d.values.ShootDeployment {
+		class := "seed"
+		if d.values.GardenDeployment {
+			class = "garden"
+		}
+		issuer.Annotations = map[string]string{source.AnnotClass: class}
+	}
```

Comment:
```suggestion
	switch (
	case d.values.GardenDeployment:
		issuer.Annotations = map[string]string{source.AnnotClass: "garden"}
	case !d.values.ShootDeployment:
		issuer.Annotations = map[string]string{source.AnnotClass: "seed"}
	)
```

Or can't we use `certClass` instead?

Reply:
Nice catch, using `certClass` is of course much simpler. Thanks!